src/desktop/apps/article/components/InfiniteScrollNewsArticle.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { Component } from 'react'
+import { flatten } from 'lodash'
+import { Article } from '@artsy/reaction/dist/Components/Publishing'
+import { ArticleData } from '@artsy/reaction/dist/Components/Publishing/Typings'
+import Waypoint from 'react-waypoint'
+import styled from 'styled-components'
+import { positronql as _positronql } from 'desktop/lib/positronql'
+import { newsArticlesQuery } from 'desktop/apps/article/queries/articles'
+import { setupFollows, setupFollowButtons } from './FollowButton.js'
+
+export interface Props {
+  article: ArticleData
+  isMobile: boolean
+  marginTop: string
+}
```

Comment:
Note that we'll only get proper type-checking between interdependent .tsx components so over time our coverage will improve. But thats for getting this started 👍